File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/changelly/ErrorBean.java
package com.androidevlinux.percy.UTXO.data.models.changelly;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by percy on 19/11/17.
 */

public class ErrorBean implements Serializable {

    @SerializedName("code")
    @Expose
    private Integer code;
    @SerializedName("message")
    @Expose
    private String message;
    private final static long serialVersionUID = 6794346013045989207L;

    public Integer getCode() {
        return code;
    }

    public void setCode(Integer code) {
        this.code = code;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/changelly/ErrorBean.kt
package com.androidevlinux.percy.UTXO.data.models.changelly

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

/**
 * Created by percy on 19/11/17.
 */

class ErrorBean : Serializable {

    @SerializedName("code")
    @Expose
    var code: Int? = null
    @SerializedName("message")
    @Expose
    var message: String? = null

    companion object {
        private const val serialVersionUID = 6794346013045989207L
    }
}
